queries/period_slippage.sql

```diff
@@ -70,18 +80,14 @@ other_transfers as (
                end            as transfer_type
     from erc20."ERC20_evt_Transfer" t
              inner join gnosis_protocol_v2."batches" b
-                        on evt_tx_hash = tx_hash
+                        on evt_tx_hash = b.tx_hash
+             inner join batchwise_traders bt
+                on evt_tx_hash = bt.tx_hash
     where b.block_time between '{{StartTime}}'
         and '{{EndTime}}'
       and '\x9008D19f58AAbD9eD0D60971565AA8510560ab41' in ("to", "from")
-      and "from" not in (
-        select trader_in
-        from filtered_trades
-    )
-      and "to" not in (
-        select trader_out
-        from filtered_trades
-    )
+      and "from" not in (select unnest(traders_in))
```

Comment:
Cool! nice.
Do we even need the select and unnest? can't we just go for
```
and "from" not in traders_in
```

Reply:
Yes we need the select and unnest. 
But we can also do this if you prefer

```sql
and not (from = any(traders_in)) 
and not (to = any(traders_out))
```